aws-cpp-sdk-core/source/utils/crypto/openssl/CryptoImpl.cpp

```diff
@@ -271,7 +286,13 @@ namespace Aws
 
             void OpenSSLCipher::Init()
             {
-                EVP_CIPHER_CTX_init(&m_ctx);
+#if OPENSSL_VERSION_NUMBER < 0x10100003L
+                m_ctx = &_m_ctx;
+                EVP_CIPHER_CTX_init(m_ctx);
```

Comment:
not sure why this is inside the compiletime ifdefs, isn't it the same between versions?